### PR TITLE
updated guidance per engineering feedback.

### PR DESCRIPTION
**Pull request by cartergreen**
updated ms-reviewer field
updated guidance for clearer results
removed create folder guidance

### CODE REVIEW

**Comment by scanum** [comment]
v

File: OneDrive/redirect-known-folders.md [hunk 1]
```diff
@@ -94,20 +94,20 @@ The OneDrive Known Folder Move Group Policy objects won't work if you previously
 - If folders have been redirected to OneDrive using Windows Folder Redirection Group Policy:
 
   1. Disable the Window Folder Redirection Group Policy and make sure to leave the folder and contents on OneDrive.
-  2. Enable KFM Group Policy. Known folder contents aren't moved.
+  2. Enable KFM Group Policy. Known folder remain in OneDrive.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:97]:
```suggestion
  2. Enable KFM Group Policy. Known folder remains in OneDrive.
```

**Comment by scanum** [comment]
@zhdong PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cartergreen** [comment]
Zhdong has left the company, that is why I've replaced her alias with mine.

**Pull request merged**